session.go

```diff
@@ -42,9 +42,9 @@ type SessionOptions struct {
 // A session multiplexes Receivers.
 type Session struct {
 	channel       uint16                       // session's local channel
-	remoteChannel uint16                       // session's remote channel, owned by conn.mux
+	remoteChannel uint16                       // session's remote channel, owned by conn.connReader
 	conn          *Conn                        // underlying conn
-	rx            chan frames.Frame            // frames destined for this session are sent on this chan by conn.mux
+	rx            chan frames.Frame            // frames destined for this session are sent on this chan by conn.connReader
```

Comment:
Is there any reason why this channel should not be buffered? I left a comment in the connReader() loop, but it seems like if a session falls behind it can clog up dispatching.

Reply:
I believe this needs to be split into two channels, one for transfer frames and one for all others.  The transfer frames channel should be buffered to the size of our incoming window.

For the non-transfer frames channel, I don't _think_ it needs to be buffered but I haven't had a chance to dig into it deeply enough.  The part that can cause clogs (or worse), and that I want to fix in general, is when the link is writing to its session mux and has to "pump" it.